### DISCUSSION TITLE
Kanji for Metaphor showing up incorrectly?

### DISCUSSION THREAD

**Discussion by zutari** [fabd/kanji-koohii#313] [3. Bugs]
It appears as 喩
![kanji](https://github.com/fabd/kanji-koohii/assets/41104342/0f92b23a-28d1-454f-91e5-50bdb9457933)

**Comment by vhagedorn** [comment]
This is the correct one, see [wikipedia's list](https://en.wikipedia.org/wiki/List_of_j%C5%8Dy%C5%8D_kanji).

Kanji typically will be the traditional ones as that's where they came from, but Heisig just decided to use the Chinese character either by mistake or because he deemed it easier to learn with the known primitives. Also, 兪 doesn't appear again in the book.
